services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> get_user_model():
+    return get_user_model().objects.get(pk=user_id)

```

Comment:
Consider adding error handling for the `get_user` function to manage cases where a user with the specified ID does not exist, such as using `get_user_model().objects.filter(pk=user_id).first()` or handling `User.DoesNotExist` exception.